src/html/classic/js/gsa_chart_helpers.js

```diff
@@ -1587,25 +1591,37 @@
   gch.default_column_label = function(info, capitalize_label,
                                   include_type, include_stat) {
     var label = '';
+    var field_name
+          = label + gch.field_name(info.column ? info.column : info.stat,
+                                   info.type);
+
     if (include_stat) {
       switch (info.stat) {
         case 'min':
-          label = label + 'min. ';
+          label = label + gsa._('min. {{field_name}}',
+                                {field_name: field_name,
+                                 interpolation: {escape: false}});
```

Comment:
After looking at https://www.i18next.com/interpolation.html did they changed `escape` to `escapeValue`? I have always used `_('min. {{- field_name}}')` until now. Did you check if this works?

Reply:
Just saw `escape` expects a function https://www.i18next.com/interpolation.html#all-interpolation-options. Therefore your changes shouldn't work atm.